DEVELOPERS.md

```diff
@@ -0,0 +1,27 @@
+# Developer Tricks
+
+Here are a few features which are useful when manually testing the game.
+
+## Logging
+
+To enable logging, create a file called `logging_config.json` in the root `cronenbroguelike` directory (i.e., a sibling to this file. This file recognizes the following parameters:
```

Comment:
Should this say "modify" rather than "create" the file?

Reply:
Ah, good catch. Fixed!